### PR TITLE
One more password auth method so we can connect to more servers

### PR DESCRIPTION
**Pull request by viklund**
Discovered that we couldn't log on to Mosler with the previous auth system, this commit fixes that.

### CODE REVIEW

File: src/main/java/se/nbis/sftp-squid/PasswordAuth.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package se.nbis.sftpsquid;
```
Comment (jhagberg) [inline src/main/java/se/nbis/sftp-squid/PasswordAuth.java:1]: se.nbis? Is that your package?

**Comment by jhagberg** [comment]
Can you describe why you needed this change?

File: src/main/java/se/nbis/sftp-squid/PasswordAuth.java (hunk 1 above)
Comment (viklund) [inline src/main/java/se/nbis/sftp-squid/PasswordAuth.java:1]: Yes, all classes need to live in a namespace, the Java Way(tm) is to reverse the domain name of the organisation as the initial namespace.

**Comment by viklund** [comment]
Mosler does not use the "KeyboardInteractive" path for authentication, it uses the "Password" path. It's part of the SSH-protocol I think, the server can present several variants for authentication, so we need to handle them.

**Comment by jhagberg** [comment]
Looks ok!

read about RFC 4252 (http://www.ietf.org/rfc/rfc4252.txt) which describes
(among other things) the mandatory password method,

**Pull request merged**